src/virt-install

```diff
@@ -88,6 +90,25 @@ ostreesetup --nogpg --osname={args.ostree_stateroot} --remote={args.ostree_remot
 %pre
 mkdir -p /mnt/ostree-repo
 mount -t 9p -o ro,trans=virtio,version=9p2000.L /mnt/ostree-repo /mnt/ostree-repo
+%end
+    """)
+# Hacky bits to copy out all of the Anaconda logs on shutdown
+if args.logs:
+    ks_tmp.write("""
+%pre
+set -euo pipefail
+mkdir -p /mnt/logs
+mount -t 9p -o rw,trans=virtio,version=9p2000.L /mnt/logs /mnt/logs
+cat >/etc/systemd/system/coreos-virtinstall-logs.service <<EOF
+[Unit]
+RequiresMountsFor=/mnt/logs
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStop=/bin/sh -c 'cp /tmp/*.log /mnt/logs'
+EOF
+systemctl daemon-reload
+systemctl start coreos-virtinstall-logs
```

Comment:
so this service does nothing on start, but on stop (shutdown) it'll copy logs to /mnt/logs/ before the mount gets unmounted? 

That's pretty cool